internal/rest/restgateway.go

```diff
@@ -275,6 +282,14 @@ func (g *RESTGateway) Start() (err error) {
 		if err = mongoStore.connect(); err != nil {
 			return
 		}
+	} else if g.conf.LevelDB.Path != "" {
```

Comment:
One more @jimthematrix, sorry.
This configuration could be confusing for users, because the `openapi` config also contains a level db.
Maybe you could provide an example of how the config would look when you've specified the:
- LevelDB for event streams state
- LevelDB for OpenAPI config
- LevelDB for receitps

... just to make sure it's clear reading the YAML.

Reply:
Retracting this comment, as I tracked this through the config and agree this is the most consistent 👍